cmd/kubeadm/app/constants/constants.go

```diff
@@ -140,6 +140,12 @@ const (
 	// MasterConfigurationConfigMapKey specifies in what ConfigMap key the master configuration should be stored
 	MasterConfigurationConfigMapKey = "MasterConfiguration"
 
+	// KubeletBaseConfigurationConfigMap specifies in what ConfigMap in the kube-system namespace the init kubelet configuration should be stored
+	KubeletBaseConfigurationConfigMap = "kubelet-base-config-1.9"
+
+	// KubeletBaseConfigurationConfigMapKey specifies in what ConfigMap key the init kubelet configuration should be stored
```

Comment:
nit on terminology, in the dynamic config jargon, the "init configuration" refers to an on-disk config file provisioned with the node, not a remote config from a ConfigMap (the file is the first config the Kubelet starts up with, and is the baseline fallback in a recovery situation). Something like "initial remote configuration" might be more appropriate for this comment.